defaults/jq/semconv.jq

```diff
@@ -0,0 +1,102 @@
+def semconv_group_attributes_by_namespace:
+    group_by(.namespace)
+    | map({ namespace: .[0].namespace, attributes: . | sort_by(.name) });
+
+#####################
+# Attribute functions
+#####################
+def semconv_attributes($options):
+    .groups
+    | map(select(.type == "attribute_group" and (.id | startswith("registry."))))
```

Comment:
not blocking this PR, but could help our future selves: 

we should have a more formal way to identify attribute definition than a group name prefix.

Reply:
I agree, but that probably implies a change in the semconv schema, or are you thinking about something else?

Reply:
I think we're talking a schema change, and I'm supportive of this.  We should enrich our model to both *simplify* and enhance it to support the concepts we need.

Reply:
Created https://github.com/open-telemetry/semantic-conventions/issues/1269 to track it, resolving the comment here